dotCMS/src/integration-test/java/com/dotcms/datagen/TestDataUtils.java

```diff
@@ -459,6 +470,8 @@ public static ContentType getNewsLikeContentType(final String contentTypeName,
                         .name(contentTypeName)
                         .velocityVarName(contentTypeName)
                         .workflowId(workflowIds)
+                        .expireDateFieldVarName(expireDateFieldName)
+                        .publishDateFieldVarName(publishDateFieldName)
```

Comment:
this util class was not creating the News like Content Type correctly because it was missing the expire and publish date Content Type's attribute